dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -27,8 +27,9 @@ def index
   end
 
   def applications_closed
+    false
```

Comment:
Could you give more context about this change?

Reply:
thank you Meg, nice catch!